Files/Scripts/Options/Ocarina of Time.psm1

```diff
@@ -3406,6 +3426,10 @@ function CreateTabColors() {
 
     CreateSpinAttackColorOptions
     CreateSwordTrailColorOptions
+    CreateReduxTextBox -Name "AlphaTip1" -Length 3  -Text "Tip Alpha Start" -Value 255 -Min 0 -Max 255 -Info "Set the starting tip transparency of the sword trail" -Credits "Anthrogi"
```

Comment:
Not a fan of this, I would rather see this and it's code changes removed

Reply:
Will show this.